parser/parser.y

```diff
@@ -1699,6 +1700,17 @@ Operand:
 		l.ParamList = append(l.ParamList, pm)
 		$$ = pm
 	}
+|	"ROW" '(' Expression ',' ExpressionList ')'
```

Comment:
Add a test cast in parser_test.go.
Is this valid syntax "row (expr)" which with no ',' in parenthesis?